crypto/multikey/kind_test.go

```diff
@@ -86,7 +86,7 @@ func TestKeyKindFromSignature(t *testing.T) {
 			"success-secp256k1-sofia-key",
 			args{
 				func() []byte {
-					k, _ := secp256k1.PrivateKeyFromHex("01901E63389EF02EAA7C5782E08B40D98FAEF835F28BD144EECF5614A415943F")
+					k, _ := secp256k1.PrivateKeyFromBytes([]byte("01901E63389EF02EAA7C5782E08B40D98FAEF835F28BD144EECF5614A415943F"))
```

Comment:
`[]byte(string)` will attempt a literal conversation not work as expected as the `[]byte` is encoded as a hex string. For testing you can use the `encodingtest.MustDecodeHex` e.g. replace `[]byte` with `encodingtest.MustDecodeHex`.